src/ProgressSnackbar.tsx

```diff
@@ -0,0 +1,112 @@
+import { ReactElement, SyntheticEvent } from "react";
+import SVG from "react-inlinesvg";
+
+import {
+  CircularProgress,
+  Typography,
+  Box,
+  Snackbar,
+  SnackbarContent,
+  ThemeProvider,
+  StyledEngineProvider,
+  Theme,
+} from "@mui/material";
+// import makeStyles from "@mui/styles/makeStyles";
+import { theme } from "./theme";
+
+export const imgSrc = (src: string, type = "svg"): string =>
+  new URL(`/src/assets/icons/${src}.${type}`, import.meta.url).href;
+
+declare module "@mui/styles/defaultTheme" {
+  // eslint-disable-next-line @typescript-eslint/no-empty-interface
+  interface DefaultTheme extends Theme {}
```

Comment:
We should be able to put this somewhere global rather than in every file?

Reply:
I have tried doing this in the theme.tsx. Where I'm I going wrong?  
This is a syntax I am unfamiliar with.  @cooper667 
<img width="955" alt="Screenshot 2022-02-22 at 12 06 34" src="https://user-images.githubusercontent.com/37402968/155129544-6e70a191-45ba-4826-817d-aa594eb452df.png">

Reply:
try creating `theme.d.ts`

and putting

```
declare module "@mui/styles/defaultTheme" {
  // eslint-disable-next-line @typescript-eslint/no-empty-interface
  interface DefaultTheme extends Theme {}
 }
 ```
  
  in it

Reply:
Yea we're limited to 3 and are already using those on a bunch of dependabot branches for some reason